packages/location-field/src/stories/Desktop.story.tsx

```diff
@@ -10,7 +10,8 @@ import {
   hereGeocoderConfig,
   layerColorMap,
   onLocationSelected,
-  selectedLocation
+  selectedLocation,
+  intlDecorator
```

Comment:
Sort imports.

Reply:
Addressed in 18d5606109e7dfce7d00d42b42c8b1b6f075a0bc